configure.ac

```diff
@@ -57,10 +57,40 @@ AC_ARG_WITH([rootlibdir],
         [], [with_rootlibdir=$libdir])
 AC_SUBST([rootlibdir], [$with_rootlibdir])
 
+AC_ARG_WITH([systemdsystemunitdir],
+			AC_HELP_STRING([--with-systemdsystemunitdir=DIR],
+			[path to systemd system unit directory]),
+					[path_systemunitdir=${withval}])
+if (test "${enable_systemd}" != "no" && test -z "${path_systemunitdir}"); then
+	AC_MSG_CHECKING([systemd system unit dir])
+	path_systemunitdir="`$PKG_CONFIG --variable=systemdsystemunitdir systemd`"
```

Comment:
Oh, indeed there's another message about the missing flag - but not stating the flag itself, maybe that can be improved? Something like 'Please use ´--with-systemdsystemunitdir=´? 

Also, update README and configure example showed when one runs `./autogen.sh`?